### PR TITLE
Fog/Orchestration Docs

### PR DESCRIPTION
**Pull request by mdarby**
Fleshed out Fog/Orchestration sample code

### CODE REVIEW

File: src/docs/orchestration/getting-started/samples/create_stack.rst [hunk 1]
```diff
@@ -20,7 +20,13 @@
 
 .. code-block:: ruby
 
-  # Not currently supported by this SDK
+  redis_template = File.read("redis_template.yml")
```
Comment (rdodev) [inline src/docs/orchestration/getting-started/samples/create_stack.rst]: Generally we don't provide this type of context in the getting-started guides. Template can come can come from a number of sources, so for the sake of clarity just introduce the variable in the call.

**Comment by rdodev** [comment]
:+1:

**Comment by rdodev** [comment]
@smashwilson can you give this a quick look and merge when you get a chance?

File: src/docs/orchestration/getting-started/samples/create_stack.rst [hunk 2]
```diff
@@ -20,7 +20,10 @@
 
 .. code-block:: ruby
 
-  # Not currently supported by this SDK
+  stack = @client.stacks.new.save(
```
Comment (smashwilson) [inline src/docs/orchestration/getting-started/samples/create_stack.rst]: Does `@client.stacks.create` work? It'd be a tiny bit shorter that way.

File: src/docs/orchestration/getting-started/samples/delete_stack.rst [hunk 3]
```diff
@@ -20,7 +20,7 @@
 
 .. code-block:: ruby
 
-  # Not currently supported by this SDK
+  @client.stacks.get({stack_id}).delete
```
Comment (smashwilson) [inline src/docs/orchestration/getting-started/samples/delete_stack.rst]: In other guides, I've written them as though they were part of the same context, and used instance variables to mark variables that "persist" from one snippet to the next. I think it provides a subtle hint to readers about where they might find a stack id, for example.

So the "create stack" sample up above might become:

```ruby
@stack = @client.stacks.create( ... )
```

And this one would be:

```ruby
@client.stacks.get(@stack.id).delete
```

Although you could likely also do:

```ruby
@stack.delete
```

File: src/docs/orchestration/getting-started/samples/get_stack_data.rst [hunk 4]
```diff
@@ -20,7 +20,7 @@
 
 .. code-block:: ruby
 
-  # Not currently supported by this SDK
+  @client.stacks.get({stack_name}, {stack_id})
```
Comment (smashwilson) [inline src/docs/orchestration/getting-started/samples/get_stack_data.rst]: Hmm! Do you need both name and ID to get a specific stack, or does the method accept either...?

Comment (mdarby) [inline src/docs/orchestration/getting-started/samples/get_stack_data.rst]: Unfortunately, yes both are necessary. :(

File: src/docs/orchestration/getting-started/samples/delete_stack.rst (hunk 3 above)
Comment (mdarby) [inline src/docs/orchestration/getting-started/samples/delete_stack.rst]: Good pointer!

File: src/docs/orchestration/getting-started/samples/get_stack_data.rst (hunk 4 above)
Comment (smashwilson) [inline src/docs/orchestration/getting-started/samples/get_stack_data.rst]: Wow, weird. Carry on then!

**Comment by mdarby** [comment]
Updated with @rdodev && @smashwilson's recommendations.

**Comment by smashwilson** [comment]
:+1: :shipit:

**Pull request merged**